tensorflow/java/src/test/java/org/tensorflow/SessionTest.java

```diff
@@ -35,9 +35,10 @@ public void runUsingOperationNames() {
     try (Graph g = new Graph();
         Session s = new Session(g)) {
       TestUtil.transpose_A_times_X(g, new int[][] {{2}, {3}});
-      try (Tensor x = Tensor.create(new int[][] {{5}, {7}});
-          AutoCloseableList<Tensor> outputs =
-              new AutoCloseableList<Tensor>(s.runner().feed("X", x).fetch("Y").run())) {
+      try (@SuppressWarnings("unchecked")
+              Tensor<Integer> x = (Tensor<Integer>) Tensor.create(new int[][] {{5}, {7}});
```

Comment:
Let's avoide the use of `@SuppressWarnings` in all tests, and use `.expect()` instead